### PR TITLE
Dutch Translate

### PR DESCRIPTION
**Pull request by timnea**
I added the Dutch translation to the app.

v1.0

### CODE REVIEW

**Review by Kr0oked — CHANGES_REQUESTED** [review]
Thanks a lot for the translation!
In the latest commit a new string 'action_screen_rotation' was adeded, could you include this one in the translation?
And the string 'compass_rose_image_description'  is not translated. Its not an important one but then its complete.

**Comment by timnea** [comment]
I forgot about the line "compass_rose_image_description" but it is all translated now.

File: app/src/main/res/values-nl/strings.xml [hunk 1]
```diff
@@ -0,0 +1,86 @@
+?xml version="1.0" encoding="utf-8"?>
```
Comment (Kr0oked) [inline app/src/main/res/values-nl/strings.xml:1]: First character '<' missing.
It should be: <?xml version="1.0" encoding="utf-8"?>

**Review by Kr0oked — CHANGES_REQUESTED** [review]
I've noticed one small copy paste error. But then it's ready to submit.

**Pull request closed without merging**